lib/active_model/serializer/attributes.rb

```diff
@@ -14,10 +14,12 @@ module Attributes
 
         # Return the +attributes+ of +object+ as presented
         # by the serializer.
-        def attributes(requested_attrs = nil, reload = false)
+        def attributes(options = {}, reload = false)
+          requested_attrs = options[:only]
+          excepts = Array(options[:except])
```

Comment:
Do you think this can be made to work with the if/unless in https://github.com/rails-api/active_model_serializers/pull/1403 ? That seems to cover this case except for having options to pass in from the renderer.